network-api/networkapi/wagtailpages/templates/wagtailpages/mini_site_name_space.html

```diff
@@ -16,8 +16,4 @@
   {% endif %}
 {% endblock %}
 
-
-{% block html_content_classes %}{% if singleton_page %}col-12 col-lg-10 offset-lg-1{% else %}col-12 col-md-9{% endif %}{% endblock %}
-
-
-{% block pre_footer_classes %}{% if singleton_page %}col-12 col-lg-10 offset-lg-1{% else %}col-md-12{% endif %}{% endblock %}
+{% block pre_footer_classes %}{% if singleton_page %}col-12 col-lg-10 offset-lg-1{% else %}col-md-12{% endif %}{% endblock %}\
```

Comment:
seems like a `\` got accidentally added at the end of this line? 🙈

Reply:
very odd!

Reply:
```suggestion
{% block pre_footer_classes %}{% if singleton_page %}col-12 col-lg-10 offset-lg-1{% else %}col-md-12{% endif %}{% endblock %}
```